app/my_jss/index.js

```diff
@@ -2,3 +2,82 @@
 function loginFunction(){
   window.location.href = "landing_page.html";
 }
+var serverObject = {
+    "success": {
+        "total": 1
+    },
+    "contents": {
+        "quotes": [
+            {
+                "quote": "Be not afraid of greatness. Some are born great, some achieve greatness, and some have greatness thrust upon 'em....",
+                "length": "116",
+                "author": "William Shakespeare",
+                "tags": [
+                    "inspire"
+                ],
+                "category": "inspire",
+                "date": "2016-03-28",
+                "title": "Inspiring Quote of the day",
+                "background": "https://theysaidso.com/img/bgs/man_on_the_mountain.jpg",
+                "id": "FG7_PlYnhPFaWL79P5076QeF"
+            }
+        ]
+    }
+};
+
+var pageElement = document.getElementById('bringInfo');
+var textElement = document.getElementById('updateText');
+
+
+    $.ajax({
+      url: "http://quotes.rest/qod.json",
+      type: "GET",
+      dataType: "json",
+      success: function (result) {
+        var objectReceived = JSON.parse(serverObject);
+        console.log( objectReceived);
+      },
+      error: function (xhr, ajaxOptions, thrownError) {
+       pageElement.innerHTML = "Too Many Requests: Rate limit of 10 requests per hour exceeded. Please wait for 49 minutes and 8 seconds.";
+      }
+  });
+
+  $.ajax({
```

Comment:
Serverul nu mergea, nu aveam rezultat pe succes

On 29.03.2016 16:53, Andrei Sebastian Cîmpean wrote:
>
> In app/my_jss/index.js 
> <https://github.com/SOFTVISION-University/cluj-pre-js-2016/pull/110#discussion_r57726109>:
>
> > +
> > +
> > +    $.ajax({
> > +      url: "http://quotes.rest/qod.json",
> > +      type: "GET",
> > +      dataType: "json",
> > +      success: function (result) {
> > +        var objectReceived = JSON.parse(serverObject);
> > +        console.log( objectReceived);
> > +      },
> > +      error: function (xhr, ajaxOptions, thrownError) {
> > +       pageElement.innerHTML = "Too Many Requests: Rate limit of 10 requests per hour exceeded. Please wait for 49 minutes and 8 seconds.";
> > +      }
> > +  });
> > +
> > +  $.ajax({
>
> asta pare că nu afişează textul? oare ce încercai să faci în 
> callback-ul de succes ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/SOFTVISION-University/cluj-pre-js-2016/pull/110/files/50614d9f5b9a26441c9fb534a63ba03bdc1518fb..5366bdfa333d3281330e63b4624263509cadcf4c#r57726109>
>